src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+
+    margin: 0 0 4px;
+    padding: 0;
+    color: $titleColor;
+  }
+
+  &__product-code {
+    color: $productColor;
+    font-weight: 400;
```

Comment:
still not resolved
![image](https://github.com/mate-academy/layout_catalog/assets/55200530/16acf39a-10e7-4db9-8f55-d14ea4ad12a5)